homeassistant/components/sensor/rainbird.py

```diff
@@ -0,0 +1,81 @@
+"""
+Support for Rain Bird Irrigation system LNK WiFi Module.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/rainbird/
```

Comment:
Update link.

Reply:
The link will work with the new docs? As there will be default component docs now

Reply:
Fixed